ARCHITECTURE.md

```diff
@@ -4,16 +4,17 @@ The core function of dbt is SQL compilation and execution. Users create projects
 
 Most of the python code in the repository is within the `core/dbt` directory. Currently the main subdirectories are:
 
-- [`adapters`](core/dbt/adapters): Define base classes for behavior that is likely to differ across databases
-- [`clients`](core/dbt/clients): Interface with dependencies (agate, jinja) or across operating systems
-- [`config`](core/dbt/config): Reconcile user-supplied configuration from connection profiles, project files, and Jinja macros
-- [`context`](core/dbt/context): Build and expose dbt-specific Jinja functionality
-- [`contracts`](core/dbt/contracts): Define Python objects (dataclasses) that dbt expects to create and validate
-- [`deps`](core/dbt/deps): Package installation and dependency resolution
-- [`graph`](core/dbt/graph): Produce a `networkx` DAG of project resources, and selecting those resources given user-supplied criteria
-- [`include`](core/dbt/include): The dbt "global project," which defines default implementations of Jinja2 macros
-- [`parser`](core/dbt/parser): Read project files, validate, construct python objects
-- [`task`](core/dbt/task): Set forth the actions that dbt can perform when invoked
+- [`adapters`](core/dbt/adapters/README.md): Define base classes for behavior that is likely to differ across databases
+- [`clients`](core/dbt/clients/README.md): Interface with dependencies (agate, jinja) or across operating systems
+- [`config`](core/dbt/config/README.md): Reconcile user-supplied configuration from connection profiles, project files, and Jinja macros
+- [`context`](core/dbt/context/README.md): Build and expose dbt-specific Jinja functionality
+- [`contracts`](core/dbt/contracts/README.md): Define Python objects (dataclasses) that dbt expects to create and validate
+- [`deps`](core/dbt/deps/README.md): Package installation and dependency resolution
+- [`events`](core/dbt/events/README.md): Logging events
```

Comment:
There is no stub README for events

Reply:
That's because it already has a README.md